src/Rector/FuncCall/FactoryFuncCallToStaticCallRector.php

```diff
@@ -71,6 +83,10 @@ public function refactor(Node $node): ?Node
 
         $model = $firstArgValue->class;
 
+        if ($model instanceof Name && $this->allowList !== [] && ! in_array($model->toString(), $this->allowList, true)) {
```

Comment:
Might be good to have this in a simple method on the rule to improve readability